cirq-google/cirq_google/devices/google_noise_properties_test.py

```diff
@@ -0,0 +1,258 @@
+    assert len(noisy_circuit.moments[0].operations) == 1
+    assert noisy_circuit.moments[0].operations[0] == op.with_tags(PHYSICAL_GATE_TAG)
+
+    # Depolarizing noise
+    assert len(noisy_circuit.moments[1].operations) == 1
+    depol_op = noisy_circuit.moments[1].operations[0]
+    assert isinstance(depol_op.gate, cirq.DepolarizingChannel)
+    assert np.isclose(depol_op.gate.p, 0.00081252)
+
+    # Thermal noise
+    assert len(noisy_circuit.moments[2].operations) == 1
+    thermal_op = noisy_circuit.moments[2].operations[0]
+    assert isinstance(thermal_op.gate, cirq.KrausChannel)
+    thermal_choi = cirq.kraus_to_choi(cirq.kraus(thermal_op))
+    assert np.allclose(
+        thermal_choi,
+        [
+            [1, 0, 0, 9.99750031e-01],
+            [0, 2.49968753e-04, 0, 0],
+            [0, 0, 0, 0],
+            [9.99750031e-01, 0, 0, 9.99750031e-01],
+        ],
+    )
+
+
+@pytest.mark.parametrize(
+    'op',
+    [
+        cirq.ISWAP(*cirq.LineQubit.range(2)) ** 0.6,
+        cirq.CZ(*cirq.LineQubit.range(2)) ** 0.3,
+        cirq_google.SYC(*cirq.LineQubit.range(2)),
+    ],
+)
+def test_two_qubit_gates(op):
+    q0, q1 = cirq.LineQubit.range(2)
+    props = sample_noise_properties([q0, q1], [(q0, q1), (q1, q0)])
+    model = NoiseModelFromGoogleNoiseProperties(props)
+    circuit = cirq.Circuit(op)
+    noisy_circuit = circuit.with_noise(model)
+    assert len(noisy_circuit.moments) == 4
+    assert len(noisy_circuit.moments[0].operations) == 1
+    assert noisy_circuit.moments[0].operations[0] == op.with_tags(PHYSICAL_GATE_TAG)
+
+    # Depolarizing noise
+    assert len(noisy_circuit.moments[1].operations) == 1
+    depol_op = noisy_circuit.moments[1].operations[0]
+    assert isinstance(depol_op.gate, cirq.DepolarizingChannel)
+    assert np.isclose(depol_op.gate.p, 0.00719705)
+
+    # FSim angle corrections
+    assert len(noisy_circuit.moments[2].operations) == 1
+    fsim_op = noisy_circuit.moments[2].operations[0]
+    assert isinstance(fsim_op.gate, cirq.PhasedFSimGate)
+    assert fsim_op == PhasedFSimGate(theta=0.01, zeta=0.03, chi=0.04, gamma=0.05, phi=0.02).on(
+        q0, q1
+    )
+
+    # Thermal noise
+    assert len(noisy_circuit.moments[3].operations) == 2
+    thermal_op_0 = noisy_circuit.moments[3].operation_at(q0)
+    thermal_op_1 = noisy_circuit.moments[3].operation_at(q1)
+    assert isinstance(thermal_op_0.gate, cirq.KrausChannel)
+    assert isinstance(thermal_op_1.gate, cirq.KrausChannel)
+    thermal_choi_0 = cirq.kraus_to_choi(cirq.kraus(thermal_op_0))
+    thermal_choi_1 = cirq.kraus_to_choi(cirq.kraus(thermal_op_1))
+    expected_thermal_choi = np.array(
+        [
+            [1, 0, 0, 9.99680051e-01],
+            [0, 3.19948805e-04, 0, 0],
+            [0, 0, 0, 0],
+            [9.99680051e-01, 0, 0, 9.99680051e-01],
+        ]
+    )
+    assert np.allclose(thermal_choi_0, expected_thermal_choi)
+    assert np.allclose(thermal_choi_1, expected_thermal_choi)
```

Comment:
same comments as for the single qubit gate test

Reply:
Done.